docs/advanced-guides/cloud-in-a-box.md

```diff
@@ -45,8 +45,14 @@ There are two types of Cloud in a Box.
 
 ## Installation
 
-1. Download one of the Cloud in a Box images of type sandbox
+:::warning
```

Comment:
Not sure if it is required to place this information inside a box. Isn't it enough to just mention this in normal text?

Reply:
Removed the warning